python/pyspark/sql/pandas/types.py

```diff
@@ -128,6 +128,8 @@ def to_arrow_type(
         arrow_type = pa.timestamp("us", tz=None)
     elif type(dt) == DayTimeIntervalType:
         arrow_type = pa.duration("us")
+    elif type(dt) == TimeType:
+        arrow_type = pa.time64("ns")
```

Comment:
referring to https://arrow.apache.org/docs/python/generated/pyarrow.time64.html, to be consistent with https://github.com/apache/spark/blob/c84629ed173bc698b187dbecdc7fdda92f6c039d/sql/api/src/main/scala/org/apache/spark/sql/util/ArrowUtils.scala#L61